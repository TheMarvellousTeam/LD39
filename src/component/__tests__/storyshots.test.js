import initStoryshots from '@storybook/addon-storyshots'

initStoryshots({
    suite: 'components snapshots',
    storyKindRegex: /^((?!.*?GameAction).)*$/,
})
