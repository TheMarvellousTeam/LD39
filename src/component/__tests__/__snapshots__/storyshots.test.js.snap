// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Bank default 1`] = `
<div
  className="bda52c9f"
>
  <div
    className="_231a0d21"
  >
    $
    900
  </div>
  <div
    className="_236e4dce"
  >
    <div
      className="_498a39a2"
    >
      +
      10
    </div>
    <div
      className="cb8c1002"
    >
      -
      10
    </div>
  </div>
  <div
    className="_236e4dce"
  >
    <div
      className="_498a39a2"
    />
    <div
      className="cb8c1002"
    >
      %
    </div>
  </div>
</div>
`;

exports[`Storyshots DrillShelf default 1`] = `
<div
  className="f810b19f"
>
  <div
    className="_2b3f086a"
    onClick={[Function]}
  >
    <div
      className="_59aa290f _5494603d "
    />
  </div>
  <div
    className="_2b3f086a"
    onClick={[Function]}
  >
    <div
      className="_59aa290f aeef4ad7 _43417977"
    />
    <div
      className="_94bff221"
    >
      $100
    </div>
  </div>
  <div
    className="_2b3f086a"
    onClick={[Function]}
  >
    <div
      className="_59aa290f _575425e5 "
    />
  </div>
  <div
    className="_2b3f086a"
    onClick={[Function]}
  >
    <div
      className="_59aa290f _98a8844b _43417977"
    />
    <div
      className="_94bff221"
    >
      $500
    </div>
  </div>
</div>
`;

exports[`Storyshots GameSpeedSlider 0 1`] = `
<div
  className="_48bbd99f"
>
  <div
    className={undefined}
  >
    gameSpeed: 0
  </div>
  <input
    className={undefined}
    max={4}
    min={0}
    onChange={[Function]}
    step={1}
    type="range"
    value={0}
  />
  <div
    className="fdc2c34b"
  >
    <button
      className={undefined}
      onClick={[Function]}
    >
      &lt;
    </button>
    <button
      className={undefined}
      onClick={[Function]}
    >
      &gt;
    </button>
  </div>
</div>
`;

exports[`Storyshots GameSpeedSlider 1 1`] = `
<div
  className="_48bbd99f"
>
  <div
    className={undefined}
  >
    gameSpeed: 1
  </div>
  <input
    className={undefined}
    max={4}
    min={0}
    onChange={[Function]}
    step={1}
    type="range"
    value={2}
  />
  <div
    className="fdc2c34b"
  >
    <button
      className={undefined}
      onClick={[Function]}
    >
      &lt;
    </button>
    <button
      className={undefined}
      onClick={[Function]}
    >
      &gt;
    </button>
  </div>
</div>
`;

exports[`Storyshots GameSpeedSlider 5 1`] = `
<div
  className="_48bbd99f"
>
  <div
    className={undefined}
  >
    gameSpeed: 5
  </div>
  <input
    className={undefined}
    max={4}
    min={0}
    onChange={[Function]}
    step={1}
    type="range"
    value={4}
  />
  <div
    className="fdc2c34b"
  >
    <button
      className={undefined}
      onClick={[Function]}
    >
      &lt;
    </button>
    <button
      className={undefined}
      onClick={[Function]}
    >
      &gt;
    </button>
  </div>
</div>
`;

exports[`Storyshots World default 1`] = `
<div
  style={
    Object {
      "bottom": 0,
      "left": 0,
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    className="c781be0c"
    onClick={[Function]}
    onMouseMove={[Function]}
    onTouchMove={[Function]}
  >
    <div
      className="_5f9c75cb"
      style={
        Object {
          "transform": "translate3d(300px,250px,0)",
        }
      }
    >
      <div
        className="_2047144c"
        style={
          Object {
            "height": 770.0000000000001,
            "left": -385.00000000000006,
            "top": -385.00000000000006,
            "width": 770.0000000000001,
          }
        }
      />
      <div
        className="_2517da5d"
        style={
          Object {
            "height": 420,
            "left": -210,
            "top": -210,
            "width": 420,
          }
        }
      >
        <canvas />
      </div>
      <div
        className="b5393d59"
      >
        <div
          className="_2308d46c"
          style={
            Object {
              "transform": "rotateZ(4.084070449666731rad)translate3d(175px,0.73px,0)rotateZ(90deg)scale(0.365,0.365)",
            }
          }
        >
          <div
            className="_4cbd7d5f _06ea2792"
            onClick={[Function]}
            style={
              Object {
                "animationDuration": "0.5s",
              }
            }
          />
        </div>
        <div
          className="_82173eee"
          onClick={[Function]}
          style={
            Object {
              "transform": "rotateZ(4.084070449666731rad)translate3d(70px,0,0)rotateZ(90deg)scale(0.2701,0.2701)",
              "transitionDuration": "0.4s",
            }
          }
        >
          <div
            className="_48a756e0 _06ea2792"
            style={
              Object {
                "animationDuration": "1s",
              }
            }
          />
        </div>
        <div
          className="_2be7f7c5"
          style={
            Object {
              "transform": "rotateZ(4.084070449666731rad)translate3d(217px,0,0)rotateZ(-4.084070449666731rad)",
            }
          }
        >
          <div
            className="_4a861cbe _568e3102 "
          >
            -5
          </div>
        </div>
      </div>
      <div
        className="b5393d59"
      >
        <div
          className="_2308d46c"
          style={
            Object {
              "transform": "rotateZ(0.9424777960769379rad)translate3d(175px,0.73px,0)rotateZ(90deg)scale(0.365,0.365)",
            }
          }
        >
          <div
            className="_4cbd7d5f _06ea2792"
            onClick={[Function]}
            style={
              Object {
                "animationDuration": "0.5s",
              }
            }
          />
        </div>
        <div
          className="_82173eee"
          onClick={[Function]}
          style={
            Object {
              "transform": "rotateZ(0.9424777960769379rad)translate3d(70px,0,0)rotateZ(90deg)scale(0.2701,0.2701)",
              "transitionDuration": "0.4s",
            }
          }
        >
          <div
            className="_48a756e0 _06ea2792"
            style={
              Object {
                "animationDuration": "1s",
              }
            }
          />
        </div>
        <div
          className="_2be7f7c5"
          style={
            Object {
              "transform": "rotateZ(0.9424777960769379rad)translate3d(217px,0,0)rotateZ(-0.9424777960769379rad)",
            }
          }
        >
          <div
            className="_4a861cbe _568e3102 "
          >
            -5
          </div>
        </div>
      </div>
      <div
        className="b5393d59"
      >
        <div
          className="_2308d46c"
          style={
            Object {
              "transform": "rotateZ(8.168140899333462rad)translate3d(175px,0.73px,0)rotateZ(90deg)scale(0.365,0.365)",
            }
          }
        >
          <div
            className="_4cbd7d5f _06ea2792"
            onClick={[Function]}
            style={
              Object {
                "animationDuration": "0.5s",
              }
            }
          />
        </div>
        <div
          className="_82173eee"
          onClick={[Function]}
          style={
            Object {
              "transform": "rotateZ(8.168140899333462rad)translate3d(52.5px,0,0)rotateZ(90deg)scale(0.2701,0.2701)",
              "transitionDuration": "0.4s",
            }
          }
        >
          <div
            className="_48a756e0 _06ea2792"
            style={
              Object {
                "animationDuration": "1s",
              }
            }
          />
        </div>
        <div
          className="_2be7f7c5"
          style={
            Object {
              "transform": "rotateZ(8.168140899333462rad)translate3d(217px,0,0)rotateZ(-8.168140899333462rad)",
            }
          }
        >
          <div
            className="_4a861cbe _568e3102 "
          >
            -5
          </div>
        </div>
      </div>
      <div
        className="b5393d59"
      >
        <div
          className="_87b4696d"
          onClick={[Function]}
          style={
            Object {
              "transform": "rotateZ(11.309733552923255rad)translate3d(175px,0,0)rotateZ(90deg)scale(0.219,0.219)",
            }
          }
        >
          <div
            className="a5e77469"
            style={
              Object {
                "animationDuration": "2s",
              }
            }
          />
        </div>
        <div
          className="_2be7f7c5"
          style={
            Object {
              "transform": "rotateZ(11.309733552923255rad)translate3d(231px,0,0)rotateZ(-11.309733552923255rad)",
            }
          }
        >
          <div
            className="_4a861cbe  _25f598aa"
          >
            +
          </div>
        </div>
      </div>
      <div
        className="_9db51524"
      >
        <div
          className="ac5bf144"
        >
          <div
            className="f7dc1047"
            style={
              Object {
                "filter": "hue-rotate(0deg) saturate(90%)",
                "transform": "rotateZ(0rad) translate3d(175px,0,0) rotateZ(94.25635973338484deg) scale(0.29600000000000004,0.29600000000000004)",
              }
            }
          />
          <div
            className="f7dc1047"
            style={
              Object {
                "filter": "hue-rotate(-8deg) saturate(92%)",
                "transform": "rotateZ(0.2rad) translate3d(175px,0,0) rotateZ(94.25635973338484deg) scale(0.3386588235294118,0.3386588235294118)",
              }
            }
          />
          <div
            className="_8666518d"
            style={
              Object {
                "filter": "hue-rotate(-1.777777777777778deg) saturate(90.44444444444444%)",
                "transform": "rotateZ(5.1000000000000005rad) translate3d(175px,0,0) rotateZ(94.25635973338558deg) scale(0.30547973856209154,0.30547973856209154)",
              }
            }
          />
          <div
            className="f7dc1047"
            style={
              Object {
                "filter": "hue-rotate(-21.333333333333332deg) saturate(95.33333333333333%)",
                "transform": "rotateZ(0.6000000000000001rad) translate3d(175px,0,0) rotateZ(94.25635973338484deg) scale(0.31175686274509806,0.31175686274509806)",
              }
            }
          />
          <div
            className="_8666518d"
            style={
              Object {
                "filter": "hue-rotate(-26.666666666666664deg) saturate(96.66666666666667%)",
                "transform": "rotateZ(4.7rad) translate3d(175px,0,0) rotateZ(94.2563597333841deg) scale(0.34019607843137256,0.34019607843137256)",
              }
            }
          />
          <div
            className="_60057334"
            style={
              Object {
                "filter": "hue-rotate(-17.77777777777778deg) saturate(94.44444444444444%)",
                "transform": "rotateZ(3.3000000000000003rad) translate3d(175px,0,0) rotateZ(94.25635973338558deg) scale(0.39079738562091504,0.39079738562091504)",
              }
            }
          />
          <div
            className="_8666518d"
            style={
              Object {
                "filter": "hue-rotate(-34.66666666666667deg) saturate(98.66666666666667%)",
                "transform": "rotateZ(5.1000000000000005rad) translate3d(175px,0,0) rotateZ(94.25635973338558deg) scale(0.3828549019607844,0.3828549019607844)",
              }
            }
          />
          <div
            className="_60057334"
            style={
              Object {
                "filter": "hue-rotate(-37.333333333333336deg) saturate(99.33333333333333%)",
                "transform": "rotateZ(3.7rad) translate3d(175px,0,0) rotateZ(94.2563597333841deg) scale(0.2990745098039216,0.2990745098039216)",
              }
            }
          />
          <div
            className="_8666518d"
            style={
              Object {
                "filter": "hue-rotate(-25.77777777777778deg) saturate(96.44444444444444%)",
                "transform": "rotateZ(4.7rad) translate3d(175px,0,0) rotateZ(94.2563597333841deg) scale(0.3354562091503268,0.3354562091503268)",
              }
            }
          />
          <div
            className="f7dc1047"
            style={
              Object {
                "filter": "hue-rotate(0deg) saturate(90%)",
                "transform": "rotateZ(0.2rad) translate3d(175px,0,0) rotateZ(94.25635973338484deg) scale(0.29600000000000004,0.29600000000000004)",
              }
            }
          />
          <div
            className="_8666518d"
            style={
              Object {
                "filter": "hue-rotate(0deg) saturate(90%)",
                "transform": "rotateZ(5.1000000000000005rad) translate3d(175px,0,0) rotateZ(94.25635973338558deg) scale(0.29600000000000004,0.29600000000000004)",
              }
            }
          />
          <div
            className="f7dc1047"
            style={
              Object {
                "filter": "hue-rotate(-25.77777777777778deg) saturate(96.44444444444444%)",
                "transform": "rotateZ(0.6000000000000001rad) translate3d(175px,0,0) rotateZ(94.25635973338484deg) scale(0.3354562091503268,0.3354562091503268)",
              }
            }
          />
          <div
            className="f7dc1047"
            style={
              Object {
                "filter": "hue-rotate(-37.333333333333336deg) saturate(99.33333333333333%)",
                "transform": "rotateZ(0rad) translate3d(175px,0,0) rotateZ(94.25635973338484deg) scale(0.2990745098039216,0.2990745098039216)",
              }
            }
          />
          <div
            className="f7dc1047"
            style={
              Object {
                "filter": "hue-rotate(-34.66666666666667deg) saturate(98.66666666666667%)",
                "transform": "rotateZ(0.2rad) translate3d(175px,0,0) rotateZ(94.25635973338484deg) scale(0.3828549019607844,0.3828549019607844)",
              }
            }
          />
          <div
            className="_60057334"
            style={
              Object {
                "filter": "hue-rotate(-17.77777777777778deg) saturate(94.44444444444444%)",
                "transform": "rotateZ(3.5rad) translate3d(175px,0,0) rotateZ(94.25635973338484deg) scale(0.39079738562091504,0.39079738562091504)",
              }
            }
          />
          <div
            className="_60057334"
            style={
              Object {
                "filter": "hue-rotate(-26.666666666666664deg) saturate(96.66666666666667%)",
                "transform": "rotateZ(3.7rad) translate3d(175px,0,0) rotateZ(94.2563597333841deg) scale(0.34019607843137256,0.34019607843137256)",
              }
            }
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
